__tests__/components/Videos/Upload/index.test.tsx

```
@@ -0,0 +1,125 @@
+    (useOrbisContext as any).mockReturnValue({ 
+      isConnected: vi.fn().mockResolvedValue(true),
+      insert: vi.fn(),
+    });
+    
+    // Reset toast mock before each test
+    vi.mocked(toast.error).mockClear();
+  });
+
+  it('should allow access when all conditions are met', async () => {
+    // Mock successful authentication checks
+    (authedOnly as any).mockResolvedValue(true);
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)